validation/verifiable_presentation.go

```diff
@@ -0,0 +1,29 @@
+package validation
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdr"
+
+	"github.com/medibloc/vc-sdk/pkg/vc"
+)
+
+type didResolver interface {
+	Resolve(did string, opts ...vdr.DIDMethodOption) (*did.DocResolution, error)
+}
```

Comment:
I guess [gobind doesn't support this kinds of export of interface](https://pkg.go.dev/golang.org/x/mobile/cmd/gobind#hdr-Type_restrictions), so I had to made this private in `vc-sdk`.
(When I make this public, [`build-android` fails](https://github.com/medibloc/vc-sdk/actions/runs/4288965665/jobs/7471532744)..
Thus, I make this interface here too.. I don't like to define this duplicated interface. Any good idea? :sob:

Reply:
No worries about gobind for now. For mobile binding, I think we need to refactor the entire structure.